WardrobeCloset/Controllers/OutfitsController.cs

```diff
@@ -0,0 +1,144 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Data.Entity;
+using System.Linq;
+using System.Net;
+using System.Web;
+using System.Web.Mvc;
+using WardrobeCloset.Models;
+
+namespace WardrobeCloset.Controllers
+{
+    public class OutfitsController : Controller
```

Comment:
We did learn about subclassing called inheritance in C#. Thinking about it now it would have made a lot of sense to put all of the code in the base controller and just call it when executing for each child class. Thanks that is very helpful!